nanoc.yaml

```diff
@@ -112,7 +112,7 @@ redirects:
   /dstu2/condition:
     url: /millennium/dstu2/general-clinical/condition
   /dstu2/conformance:
-    url: /millennium/dstu2/conformance
+    url: /millennium/dstu2/conformance/conformance
```

Comment:
we have a lot of old redirects in this file that we should really strip out

we do need to add some new redirects with this PR, but they should be scoped towards what we're actually changing. for example, we are moving content at this location: https://fhir.cerner.com/millennium/dstu2/conformance/ to this location: https://fhir.cerner.com/millennium/dstu2/conformance/conformance/

so the new redirects need to be like
```
  /millennium/dstu2/conformance
    url: /millennium/dstu2/conformance/conformance
```

Reply:
Added new redirects: [Fix](https://github.com/cerner/fhir.cerner.com/pull/323/commits/2bcc448db093c8c5488198f21d9937fba354d60e#diff-aa92b0c792e8953d5d37f9aa7b5cb130R191)

Reply:
we should revert the original changes to nanoc.yaml and only add the new redirects

(we could test to see if 'nested' redirects to anything)

Reply:
we should keep all the first round of redirects actually

Reply:
I just realized what the problem was with the redirect error. It's because I was redirecting 
`/millennium/dstu2/conformance`, which is an existing page (the Conformance tab overview), to the Conformance Metadata page `/millennium/dstu2/conformance/conformance`.

Removed this redirect [here](https://github.com/cerner/fhir.cerner.com/pull/323/commits/9b7730e887d063b3819f610b97746194d830edc1#diff-aa92b0c792e8953d5d37f9aa7b5cb130L191)